### PR TITLE
Upgrade to FGT helm chart v0.2.0 - Updated readme and example configuration

### PR DESCRIPTION
**Pull request by lukasosterheider**
=> Readme and example configuration file changes only.

### CODE REVIEW

**Comment by lukasosterheider** [comment]
Will work on the secret detection issue tomorrow.

**Comment by lukasosterheider** [comment]
@tgip-work : Hi Torsten, just readme and example config changes. Please approve and merge.
Thank you!

File: charts/fgt/README.md.gotmpl [hunk 1]
```diff
@@ -156,7 +168,7 @@ Run `helm upgrade --helm` for full list of options.
     You can install into other namespace than `default` by setting the `--namespace` parameter, e.g.
 
     ```bash
-    helm upgrade my-release-name pharmaledger-imi/fgt --version={{ template "chart.version" . }} \
+    helm upgrade my-release-name your-container-registry/fgt --version={{ template "chart.version" . }} \
```
Comment (tgip-work) [inline charts/fgt/README.md.gotmpl:171]: Hi Lukas,

the repo for the helm charts is still `pharmaledger-imi`

File: charts/fgt/README.md.gotmpl [hunk 2]
```diff
@@ -167,7 +179,7 @@ Run `helm upgrade --helm` for full list of options.
     Provide the `--wait` argument and time to wait (default is 5 minutes) via `--timeout`
 
     ```bash
-    helm upgrade my-release-name pharmaledger-imi/fgt --version={{ template "chart.version" . }} \
+    helm upgrade my-release-name your-container-registry/fgt --version={{ template "chart.version" . }} \
```
Comment (tgip-work) [inline charts/fgt/README.md.gotmpl:182]: Hi Lukas,

the repo for the helm charts is still pharmaledger-imi

File: charts/fgt/README.md.gotmpl [hunk 3]
```diff
@@ -35,28 +41,23 @@ This helm chart uses Helm [hooks](https://helm.sh/docs/topics/charts_hooks/) in
...
+      credentials: |-
+        # ... content of the credentials json file ...
 
     ```
 
 2. Install via helm to namespace `default`
 
     ```bash
-    helm upgrade my-release-name pharmaledger-imi/fgt --version={{ template "chart.version" . }} \
+    helm upgrade my-release-name your-container-registry/fgt --version={{ template "chart.version" . }} \
```
Comment (tgip-work) [inline charts/fgt/README.md.gotmpl:60]: Hi Lukas,

the repo for the helm charts is still pharmaledger-imi

File: charts/fgt/README.md [hunk 4]
```diff
@@ -33,28 +39,23 @@ This helm chart uses Helm [hooks](https://helm.sh/docs/topics/charts_hooks/) in
...
+      credentials: |-
+        # ... content of the credentials json file ...
 
     ```
 
 2. Install via helm to namespace `default`
 
     ```bash
-    helm upgrade my-release-name pharmaledger-imi/fgt --version=0.1.6 \
+    helm upgrade my-release-name your-container-registry/fgt --version=0.2.0 \
```
Comment (tgip-work) [inline charts/fgt/README.md:58]: Hi Lukas,

the repo for the helm charts is still pharmaledger-imi

File: charts/fgt/README.md [hunk 5]
```diff
@@ -154,7 +166,7 @@ Run `helm upgrade --helm` for full list of options.
     You can install into other namespace than `default` by setting the `--namespace` parameter, e.g.
 
     ```bash
-    helm upgrade my-release-name pharmaledger-imi/fgt --version=0.1.6 \
+    helm upgrade my-release-name your-container-registry/fgt --version=0.2.0 \
```
Comment (tgip-work) [inline charts/fgt/README.md:169]: Hi Lukas,

the repo for the helm charts is still pharmaledger-imi

File: charts/fgt/README.md [hunk 6]
```diff
@@ -165,7 +177,7 @@ Run `helm upgrade --helm` for full list of options.
     Provide the `--wait` argument and time to wait (default is 5 minutes) via `--timeout`
 
     ```bash
-    helm upgrade my-release-name pharmaledger-imi/fgt --version=0.1.6 \
+    helm upgrade my-release-name your-container-registry/fgt --version=0.2.0 \
```
Comment (tgip-work) [inline charts/fgt/README.md:180]: Hi Lukas,

the repo for the helm charts is still pharmaledger-imi

**Review by tgip-work — CHANGES_REQUESTED** [review]
Hi Lukas,

the repo for the helm charts is still pharmaledger-imi

**Review by tgip-work — APPROVED** [review]

**Pull request merged**